src/main.rs

```diff
@@ -277,12 +277,23 @@ async fn main() -> std::io::Result<()> {
     let tx_commitment = CommitmentConfig::from_str(&config.tx_commitment)
         .expect("one of: processed | confirmed | finalized");
     let extra_rpcs = config.extra_rpcs.as_ref();
+
+    let sub_account_ids: Vec<u16> = if config.default_sub_account_id.is_some() {
+        vec![config.default_sub_account_id.unwrap()]
+    } else {
+        config
+            .active_sub_accounts
+            .split(",")
+            .map(|s| s.parse::<u16>().unwrap())
+            .collect()
+    };
```

Comment:
this doesn't break the current behaviour
```suggestion
        let sub_account_ids = vec![config.default_sub_account_id.unwrap()].extend(
                config
            .active_sub_accounts
            .split(",")
            .map(|s| s.parse::<u16>().unwrap())
           );
```

Reply:
we just need to deduplicate right?